### PR TITLE
Adds pattern for a single response. [Amends #2611]

### PR DESCRIPTION
**Pull request by humancompanion-usds**

### CODE REVIEW

**Review by danbrady — APPROVED** [review]

File: src/_patterns/ask-users-for/a-single-response.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+7. **Simplified branching and conditional logic.** Less complex pages are easier to code for conditional branches.
+8. **Easier for screen readers.** Less complex pages are easier for users of screen readers to use.
+9. **Improved pattern reuse.** Simple pages make pattern reuse more likely and thus simplify the design process.
+10. **Simpler screens despite more clicks.** While breaking forms into smaller chunks does result in more clicks, research both here and [in the UK](https://designnotes.blog.gov.uk/2014/07/14/things-we-learnt-designing-register-to-vote/) has shown that the process feels simple and easy and users do not get stuck.  
+
+## Examples
+
+### Annotated
+
+{% include component-example.html alt="An annotated example of using radio buttons to collect the answer to a single question." file="/images/patterns/ask-users-for/a-single-response/single-response-annotated.png" caption="An annotated example of using radio buttons to collect the answer to a single response." class="x2" %}
```
Comment (Andrew565) [inline src/_patterns/ask-users-for/a-single-response.md:55]: The caption says "a single response" but the alt text says "a single question", unsure if this is intentional.

File: src/_patterns/ask-users-for/a-single-response.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+### Other relevant examples
+
+* [gov.uk Service Manual - Structuring forms](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page)
+* [Cognitive Accessibility Design Pattern: Make the Purpose of Your Page Clear](https://www.w3.org/WAI/WCAG2/supplemental/patterns/o1p01-clear-purpose/)
+* [Cognitive Accessibility Design Pattern: Avoid Too Much Content](https://www.w3.org/WAI/WCAG2/supplemental/patterns/o5p03-manageable-quantity/)
+* [The Question Protocol: How to make sure every form field is necessary](https://www.uxmatters.com/mt/archives/2010/06/the-question-protocol-how-to-make-sure-every-form-field-is-necessary.php)
+
+## How to design and build
+
+### Anatomy and layout details
```
Comment (Andrew565) [inline src/_patterns/ask-users-for/a-single-response.md:77]: This section is blank, unsure if intentional.

**Review by Andrew565 — APPROVED** [review]
Couple question marks but looks fine otherwise.

File: src/_patterns/ask-users-for/a-single-response.md (hunk 1 above)
Comment (humancompanion-usds) [reply src/_patterns/ask-users-for/a-single-response.md]: Good catch, thanks!

File: src/_patterns/ask-users-for/a-single-response.md (hunk 2 above)
Comment (humancompanion-usds) [reply src/_patterns/ask-users-for/a-single-response.md:77]: Added a pointer to the right spots.

[the remark this answers is not in the document]
Comment (humancompanion-usds) [reply src/_patterns/ask-users-for/a-single-response.md:77]: I added a pointer both up to the annotated examples and down to the components to use.

**Review by jeana-adhoc — APPROVED** [review]
No changes! I did upload updated screenshots of the two examples for ask-users-for-a-single-response.

**Pull request merged**